### ISSUE TITLE
library not found for -lRNFetchBlob

### ISSUE THREAD

**Issue by sahil290791** [joltup/rn-fetch-blob#438]
React Native: 0.60.5
Platform: iOS
I have upgraded from react-native-fetch-blob to rn-fetch-blob@0.10.16.

<img width="1008" alt="Screenshot 2019-09-15 at 5 50 54 AM" src="https://user-images.githubusercontent.com/7287280/64915106-226d8f00-d77d-11e9-903d-256e69bd266c.png">

The build fails with linker error:
```
ld: library not found for -lRNFetchBlob
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by sahil290791** [comment]
I have removed rn-fetch-blob and react-native-cached-image which was internally using react-fetch-blob.
I have also unlinked rn-fetch-blob, removed it from **package.json**, have deleted node_modules, did yarn install.
Cleaned the build folder in XCode but still the build fails with **lRNFetchBlob not found**.

I have even checked node_modules folder there is no rn-fetch-blob present but still xcode is trying to link this package. What else could I be missing?
